docs/developer-guide/coordinate-systems.md

```diff
@@ -2,11 +2,9 @@
 
 In most deck.gl layers, every data object is expected to contain one or more **positions** (e.g. the center of a point, or the start and end of an arc, the vertices of a polygon, etc). Positions are expected to be supplied as two or three element arrays (`[x, y]` or `[x, y, z]`). Objects can also have **dimensions** (e.g. the radius of a circle, the width of a path, etc), most of which are specified as a single number. Positions and dimensions are used to construct a 3D world with your data.
 
-By default, deck.gl layers interprets positions as geospatial coordinates (i.e. `[longitude, latitude, altitude]`), and dimensions as in meters. So when working with geospatial data sources such as GeoJSON, deck.gl will automatically interpret any positions correctly.
+deck.gl supports a selection of coordinate systems, addressing diverse geospatial and non-geospatial [use cases](#world-space). Each layer can set its own `coordinateSystem` prop to specify how its positions should be interpreted. Layers using different coordinate systems can be composed together, which is very useful when dealing with datasets from different sources.
```

Comment:
I'm a little surprised the `use cases` link points to `#world-space`. Is that intentional?